pad_crm/crm_lead.xml

```diff
@@ -1,22 +1,24 @@
-<openerp>
-    <data>
-        <record id="crm_case_form_view_oppor" model="ir.ui.view">
-            <field name="model">crm.lead</field>
-            <field name="inherit_id" ref="crm.crm_case_form_view_oppor"/>
-            <field name="arch" type="xml">
-                <field name="description" position="replace">
-                    <field name="description_pad" widget="pad"/>
-                </field>
-            </field>
-        </record>
-        <record id="crm_case_form_view_leads" model="ir.ui.view">
-            <field name="model">crm.lead</field>
-            <field name="inherit_id" ref="crm.crm_case_form_view_leads"/>
-            <field name="arch" type="xml">
-                <field name="description" position="replace">
-                    <field name="description_pad" widget="pad"/>
-                </field>
-            </field>
-        </record>
-    </data>
-</openerp>
+<?xml version="1.0" encoding="utf-8"?>
+<!--Copyright 2016 Ivan Yelizariev <https://it-projects.info/team/yelizariev>
+    Copyright 2019 Artem Rafailov <https://it-projects.info/team/Ommo73/>
+    License LGPL-3.0 (https://www.gnu.org/licenses/lgpl.html).-->
+<odoo>
+    <record id="crm_case_form_view_oppor" model="ir.ui.view">
```

Comment:
Don't do excessive changes, keep the commit history clean. 
you've changed `openerp data` to `odoo` so let the other inner tags be on the places they were. Also, it's hard to determine you did some changes there or just moved the block